src/System.Linq.Expressions/src/System/Linq/Expressions/Expression.netstandard1.7.cs

```diff
@@ -0,0 +1,476 @@
+    {
+        /// <summary>
+        /// Creates a <see cref="DynamicExpression" /> that represents a dynamic operation bound by the provided <see cref="CallSiteBinder" />.
+        /// </summary>
+        /// <param name="binder">The runtime binder for the dynamic operation.</param>
+        /// <param name="returnType">The result type of the dynamic expression.</param>
+        /// <param name="arguments">The arguments to the dynamic operation.</param>
+        /// <returns>
+        /// A <see cref="DynamicExpression" /> that has <see cref="NodeType" /> equal to
+        /// <see cref="ExpressionType.Dynamic">Dynamic</see> and has the
+        /// <see cref="DynamicExpression.Binder">Binder</see> and
+        /// <see cref="DynamicExpression.Arguments">Arguments</see> set to the specified values.
+        /// </returns>
+        /// <remarks>
+        /// The <see cref="DynamicExpression.DelegateType">DelegateType</see> property of the
+        /// result will be inferred from the types of the arguments and the specified return type.
+        /// </remarks>
+        public static DynamicExpression Dynamic(CallSiteBinder binder, Type returnType, IEnumerable<Expression> arguments)
+        {
+            ContractUtils.RequiresNotNull(arguments, "arguments");
+            ContractUtils.RequiresNotNull(returnType, "returnType");
+
+            var args = arguments.ToReadOnly();
+            ContractUtils.RequiresNotEmpty(args, "args");
+            return MakeDynamic(binder, returnType, args);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="DynamicExpression" /> that represents a dynamic operation bound by the provided <see cref="CallSiteBinder" />.
+        /// </summary>
+        /// <param name="binder">The runtime binder for the dynamic operation.</param>
+        /// <param name="returnType">The result type of the dynamic expression.</param>
+        /// <param name="arg0">The first argument to the dynamic operation.</param>
+        /// <returns>
+        /// A <see cref="DynamicExpression" /> that has <see cref="NodeType" /> equal to
+        /// <see cref="ExpressionType.Dynamic">Dynamic</see> and has the
+        /// <see cref="DynamicExpression.Binder">Binder</see> and
+        /// <see cref="DynamicExpression.Arguments">Arguments</see> set to the specified values.
+        /// </returns>
+        /// <remarks>
+        /// The <see cref="DynamicExpression.DelegateType">DelegateType</see> property of the
+        /// result will be inferred from the types of the arguments and the specified return type.
+        /// </remarks>
+        public static DynamicExpression Dynamic(CallSiteBinder binder, Type returnType, Expression arg0)
+        {
+            ContractUtils.RequiresNotNull(binder, "binder");
+            ValidateDynamicArgument(arg0, nameof(arg0));
+
+            System.Linq.Expressions.Compiler.DelegateHelpers.TypeInfo info = System.Linq.Expressions.Compiler.DelegateHelpers.GetNextTypeInfo(
```

Comment:
That using was already there, it's just that this type was ambiguous between System.Linq.Expressions.Compiler.DelegateHelpers and System.Dynamic.Runtime.DelegateHelpers, but I've addressed this by using an alias.